deploy/charts/preflight/templates/agent_installation.yaml

```diff
@@ -0,0 +1,214 @@
+kind: ClusterRoleBinding
+metadata:
+  name: jetstack-secure-agent-cert-manager-reader
+  labels:
+    {{- include "preflight.labels" . | nindent 4 }}
+roleRef:
+  kind: ClusterRole
+  name: jetstack-secure-agent-cert-manager-reader
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: agent
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: jetstack-secure-agent-googlecas-reader
+  labels:
+    {{- include "preflight.labels" . | nindent 4 }}
+roleRef:
+  kind: ClusterRole
+  name: jetstack-secure-agent-googlecas-reader
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: agent
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: jetstack-secure-agent-get-webhooks
+  labels:
+    {{- include "preflight.labels" . | nindent 4 }}
+roleRef:
+  kind: ClusterRole
+  name: jetstack-secure-agent-get-webhooks
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: agent
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: jetstack-secure-agent-node-reader
+  labels:
+    {{- include "preflight.labels" . | nindent 4 }}
+roleRef:
+  kind: ClusterRole
+  name: jetstack-secure-agent-node-reader
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: agent
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: jetstack-secure-agent-cluster-viewer
+  labels:
+    {{- include "preflight.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+- kind: ServiceAccount
+  name: agent
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: agent
+  labels:
+    {{- include "preflight.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: agent
+      {{- include "preflight.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "preflight.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: agent
+      volumes:
+      - name: config
+        configMap:
+          name: agent-config
+      - name: credentials
+        secret:
+          secretName: agent-credentials
+      containers:
+      - name: agent
+        image: "{{ .Values.image.repository }}:v{{ .Chart.AppVersion }}"
+        args:
+        - "agent"
+        - "-c"
+        - "/etc/jetstack-secure/agent/config/config.yaml"
+        - "-k"
+        - "/etc/jetstack-secure/agent/credentials/credentials.json"
+        - "-p"
+        - "0h1m0s"
```

Comment:
What is `-p`? Just wondering 😅

Reply:
:shrug: I just copied it from the downloaded agent_installation file. Perhaps it's the frequency of cluster scanning?